ocfweb/docs/docs/services/hpc.md

```diff
@@ -1,65 +1,115 @@
 
-### Slurm
+## Slurm
 
-We currently use [Slurm][slurm] as our workload manager for the cluster. We will soon post technical documentation about our Slurm configuration, but briefly, Slurm is a free and open-source job scheduler which helps manage multi-user use of distributed computational resources, referred to as nodes. In short, all of your programs will be run through Slurm. To use Slurm there are several commands that will be helpful:
+We currently use [Slurm][slurm] as our workload manager for the cluster. We will soon post technical documentation about our Slurm configuration, but briefly, Slurm is a free and open-source job scheduler which helps distribute jobs from all users evenly among HPC computers, referred to as nodes. In short, all of your programs will be run through Slurm. To use Slurm there are several commands that will be helpful:
 
 * `srun`: Used to submit jobs.
 * `scontrol`: Used to view and edit the Slurm configuration.
 * `squeue`: Used to view running and queued jobs.
 
-### Dependencies
-For managing application dependencies, one currently has two options:
+## Dependencies
+For managing application dependencies, you currently have two options:
 
-First you could use a virtual environment if you are using Python packages. To create a virtual environment navigate to your home directory and run the following commands
+### Virtual Environments
 
-    virtualenv -p python3.5 venv
-    . venv/bin/activate
+First you could use a [virtual environment][venv] if you are using Python packages. To create a virtual environment navigate to your home directory and run the following commands:
+
+```
+virtualenv --system-site-packages -p python3 venv
+. venv/bin/activate
+```
 
 This will allow you to *pip install* any Python packages that the OCF does not already have for your program.
 
-For those who need access to non-python dependencies or have already integrated their program into Docker, the second option is to use Singularly containers. [Singularity][singularity] is a containerization platform developed at Laurence Berkeley National Laboratory that is designed specifically for HPC environments. To read more about the benefits of Singularity you can look [Here][singularity_article]. Using a container is quite simple, below are a few commands to help getting started.
+### Singularity
+
+For those who need access to non-Python dependencies or have already integrated their program into Docker, the second option is to use Singularity containers. [Singularity][singularity] is a containerization platform developed at Lawrence Berkeley National Laboratory that is designed specifically for HPC environments. To read more about the benefits of Singularity you can look [here][singularity_article]. We suggest a particular workflow, which will help simplify deploying your program on our infrastructure.
+
+#### Installing
+
+If you are running linux on your system, you can install Singularity from the official apt repos:
```

Comment:
I would say GNU/Linux here. Even if you won't, you should still capitalize it...